src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -270,8 +270,8 @@ private void requestEntityToBuildingIfRequired(Template.EntityInfo entityInfo)
                     {
```

Comment:
No we should not.
As per MC and Forge defined operations on an Empty ItemStack are not defined, properly.
(Retrieving the Item from the Registry will fail).
We should prevent all interaction with Empty stacks if at all possible.